packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -479,6 +485,45 @@ async function handler(
     ) {
       throw new HttpError({ statusCode: 403, message: ErrorCode.CancelledBookingsCannotBeRescheduled });
     }
+    const userReschedulingIsOwner = isUserReschedulingOwner(userId, originalRescheduledBooking?.user?.id);
+    let isTeamOwnerOrAdmin = false;
+    if (isTeamEventType) {
+      const teamOwnerOrAdmin = await prisma.membership.findFirst({
+        where: {
+          teamId: eventType?.teamId || 0,
```

Comment:
In case there is no teamId on eventType, we won't be fetching any team with `teamId = 0`, so there is no point in querying the database. 
It's better to update the if statement in this case:
`    if (isTeamEventType && eventType.teamId) {`

Reply:
Done